services/core-api/app/api/mines/mine/models/mine.py

```diff
@@ -162,6 +171,27 @@ def has_minespace_users(self):
             MinespaceUserMine.mine_guid == self.mine_guid).count()
         return count > 0
 
+    @hybrid_property
+    def mine_work_information(self):
+        if self.mine_work_informations:
+            return self.mine_work_informations[0]
+        return None
+
+    @hybrid_property
+    def work_status(self):
+        if self.mine_work_informations:
+            return self.mine_work_informations[0].mine_work_status_code
+        return "UNKNOWN"
```

Comment:
"UNKNOWN" is not a valid status for BE. Is there any reason not to return None and handle it on FE as "UNKNOWN" or something else if needed?